sdk/samples/iot/hub/src/paho_iot_hub_twin_example.c

```diff
@@ -535,30 +535,43 @@ static int send_reported_property(az_span reported_property_payload)
 static az_result update_property(az_span desired_payload)
 {
   // Parse desired property payload
-  az_json_parser json_parser;
-  az_json_token token;
-  az_json_token_member token_member;
+  az_json_parser json_parser = { 0 };
```

Comment:
@momuno, please review this change to the sample. I tried to follow the same behavior from https://github.com/Azure/azure-sdk-for-c/pull/828, but this could benefit from a second look.